dev/integration_tests/flutter_gallery/lib/demo/colors_demo.dart

```diff
@@ -78,7 +78,7 @@ class ColorItem extends StatelessWidget {
   }
 }
 
-class PaletteTabView extends StatelessWidget {
+class PaletteTabView extends StatefulWidget {
```

Comment:
Since this is currently broken at head, and not related to this change, this should probably be broken out into a separate PR and validated with a test so it does regress in the future.